lib/src/util/posix/dynamic_library.cc

```diff
@@ -41,7 +41,11 @@ namespace facter { namespace util {
         close();
 
         // Don't actually perform a load to determine if it is already loaded
+#ifdef _AIX
+        _handle = dlopen(name.c_str(), RTLD_LAZY);
```

Comment:
Sounds fine to me; I'd probably prefer a comment explicitly linking it to FACT-891.